packages/solana/lib/src/rpc_client/rpc_client.dart

```diff
@@ -148,15 +149,16 @@ class RPCClient {
   ///
   /// [see this document]: https://docs.solana.com/developing/clients/jsonrpc-api#configuring-state-commitment
   Future<SimulateTxResult> simulateTransaction(
-    SignedTx signedTx, {
+    String transaction, {
     Commitment? commitment,
+    EncodingType encoding = EncodingType.base58,
```

Comment:
Why do we need to support `base58` encoding if it's deprecated?

Reply:
I see that eventhough deprecated it is still the default. Since this will now be THE library, I think we must support this. Perhaps mark it as deprecated would be the correct way.

Reply:
Let's discuss it in discord then.

Reply:
Updated!